fast64_internal/oot/oot_operators.py

```diff
@@ -51,6 +51,7 @@ def execute(self, context):
         emptyObj.name = "Door Actor"
         emptyObj.ootTransitionActorProperty.actor.actorID = "ACTOR_DOOR_SHUTTER"
         emptyObj.ootTransitionActorProperty.actor.actorParam = "0x0000"
+        emptyObj.fast64.oot.version = emptyObj.fast64.oot.cur_version
```

Comment:
I thought about this and what if the user creates an empty and sets its type manually?

Maybe it'd be a better idea to set the default value of the version prop to cur_version, that'd handle all new objects :thinking: 
But then it would mark "up-to-date" really old objects in really old blends that never had this property Sadge

idk...

Reply:
if someone adds an empty manually then it'd be set the next time they'll open the blend I think